src/pages/registration/FilesAndLicensePanel.tsx

```diff
@@ -175,6 +175,7 @@ export const FilesAndLicensePanel = ({ uppy }: FilesAndLicensePanelProps) => {
                 </Box>
                 {!categorySupportsFiles && (
                   <InfoBanner
+                    type="info"
```

Comment:
Går eventuelt an å gjøre denne typen til default siden den heter InfoBanner

Reply:
Ja, var inne på tanken :thinking: Men tenkte også at det gjør det litt lettere å finne den komponenten man leter etter i koden. Tenkte egentlig å rename komponenten, men fant ikke noe som ga mer mening enn `InfoBanner`, men er veldig åpen for forslag!